src/mongocxx/options/create_collection.cpp

```diff
@@ -74,6 +78,10 @@ bsoncxx::document::value create_collection::to_document() const {
         doc << "noPadding" << *_no_padding;
     }
 
+    if (_validation) {
+        doc << bsoncxx::builder::stream::concatenate{_validation->to_document()};
```

Comment:
Yes, I think this is actually where I was running into issues... maybe I didn't remember to push my latest patch.